src/main/java/com/bootcamp/demo/service/IUserService.java

```diff
@@ -0,0 +1,18 @@
+package com.bootcamp.demo.service;
+
+import com.bootcamp.demo.model.User;
+
+import java.util.Set;
+
+public interface IUserService {
```

Comment:
It does not seem like this is needed (especially since it is only implemented, but not used for injection). See this thread https://github.com/nymphadora-tonks-team/booking-jb/pull/20#discussion_r757110327 , which highlights more details related to this.

Reply:
^ Do not mark the above as resolved if possible since I have referenced it in another PR & it might not redirect properly afterwards 🙂